server/functions/src/index.ts

```diff
@@ -104,7 +106,7 @@ export const putClientSettings = functions
 // Method that runs when client settings have been updated, and will make those
 // updated settings take effect.
 export const clientSettingsUpdated = functions
-  .region("europe-west6")
+  .region(SERVING_REGION)
```

Comment:
Is there a way to have a test.... even a deploy time test that barfs if the TypeScript and Terraform regions don't match? They always should but I want to be careful of someone making a stupid mistake.

Reply:
I've added a predeploy test that checks the Cloud Functions region against the region Firestore has been set up for in the project being deployed to.